apache/_docroot.shtml

```diff
@@ -1,29 +1,28 @@
 <div class="section">
     <h2>Can Tugboat find your document root?</h2>
     <p>
-        Tugboat looks for a folder named
-        <tt>docroot</tt> in your repository to serve content from. This is a common pattern, but many projects have their own
-        preference. If your document root is in a different location you have a couple of different options.
+        Tugboat looks for a folder named <tt>docroot</tt> in your repository to serve content from. This is a 
+        common pattern, but many projects have their own preference. If your document root is in a different location 
+        you have a couple of different options.
     </p>
     <h3>Option 1. Add a <tt>docroot/</tt> folder to your repository</h3>
     <p>
       This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been 
-      deployed elsewhere. If you choose this option, create a <tt>docroot</tt> folder and move your existing content 
-      you want Apache to serve into that folder. Be sure to follow up on other deployments to make sure things still 
-      work.
+      deployed elsewhere. If you choose this option, create a <tt>docroot</tt> folder for Tugboat to look in. If your 
+      site is deployed elsewhere, make sure each deployment still works as expected.
     </p>
     <p>
       <!--#set var="dashboard" value="https://${TUGBOAT_DASHBOARD}/${TUGBOAT_PREVIEW_ID}" -->
       Remember to rebuild the <a href="<!--#echo var="dashboard" -->">preview</a> after you make any changes to your 
       build script. This will happen automatically for previews built for pull requests (or merge requests), but needs 
       to be done manually for other previews.
     </p>
-    <h3>Option 2. Create a build script.</h3>
+    <h3>Option 2. Create a build script</h3>
     <p>
-      Build scripts can be used to perform any number of customizations to your preview. But, if all you need is to 
-      point Apache to a different location in your repository, a simple script like the following may be all you need. 
-      Add this to a file named <code>Makefile</code> in the root of your repository. Make sure the indent is a TAB, 
-      and not multiple spaces.
+      Build scripts can be used to perform any number of customizations to your preview. But, if all you need is for 
+      Tugboat to look in a different folder within your repository, a simple script like the following may be all 
```

Comment:
Let's change this so "Tugboat" is the subject.

    But, if Tugboat just needs to look in a different folder in your repository...